src/Testing.jl

```diff
@@ -0,0 +1,38 @@
+module Testing
+
+function get_bio_fmt_specimens()
+    path = joinpath(dirname(dirname(@__FILE__)), "BioFmtSpecimens")
+    if !isdir(path)
+        run(`git clone --depth 1 https://github.com/BioJulia/BioFmtSpecimens.git $(path)`)
+    end
+end
```

Comment:
We could do releases of BioFmtSpecimens? Then we can checkout specific versions of the resource?